README.md

```diff
@@ -33,8 +33,8 @@
 <p align="center">
   <a href="#installation">Installation</a> •
   <a href="#quickstart">Quickstart</a> •
-  <a href="#datasets-28">Datasets</a> •
-  <a href="#models-30">Models</a> •
+  <a href="#datasets-27">Datasets</a> •
```

Comment:
the number of datasets decreased? I guess you have to reinstall the package beforehand.